app/models/gobierto_data/dataset.rb

```diff
@@ -43,6 +57,14 @@ def available_formats
       [:csv]
     end
 
+    def dataset_path
+      url_helpers.gobierto_data_datasets_path(id: slug)
+    end
+
+    def dataset_url
+      url_helpers.gobierto_data_datasets_url(id: slug, only_path: true)
```

Comment:
`only_path: true` returns a relative path, not the whole url, but the name of the method suggests you what the whole url (including host). Please remove only_path or clarify

Reply:
if remove only_path this method fails when reindex and during the search with:
`ArgumentError: Missing host to link to! Please provide the :host parameter, set default_url_options[:host]`

Can I remove dataset_url? and use dataset_path instead because, it is not in use from now.